wwwapp/migrations/0059_migrate_away_from_python_allaccess.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-03-07 06:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from allaccess.models import AccountAccess
```

Comment:
Never import models in migrations directly. Use proper method to get the versioned model.

https://docs.djangoproject.com/en/3.0/topics/migrations/#data-migrations